lib/src/storyboard_controller.dart

```diff
@@ -246,7 +249,7 @@ class StoryBoardController {
     final byteList = bytes.buffer.asUint8List();
     if (isCI()) {
       print(
-          "Storyboard: ${view.widget.graphForCiAuto?.relationDescription ?? "graphForCiAuto is null"}");
+          "Storyboard: ${view.widget.graphForCiAuto?.imageNumber} : ${view.widget.graphForCiAuto?.relationDescription ?? "graphForCiAuto is null"}");
```

Comment:
https://github.com/flitsoft/coding-standard/blob/master/CODING-STANDARDS.md#dont-add-logic-in-print-statement